build-tests/api-documenter-test/etc/api-documenter-test.api.json

```diff
@@ -166,7 +166,8 @@
         "@preapproved": true,
         "@docCategory": true,
         "@myCustomTag": true
-      }
+      },
+      "reportUnsupportedHtmlElements": false
```

Comment:
What is this? I can't find it anywhere in the repo.

Reply:
Good Q: this started being populated once I did the merge yesterday to resolve a conflict, so I was under the impression this was a recently included feature.

Reply:
@octogonz Ideas? @mrshllstock, if you rebase your branch off the latest changes from main, does this go away?

Reply:
I rebased main (boy was that an ugly PR state for a moment). We can see this has been added in other tests from api-documenter, such as the tsdocConfig test.

Reply:
Ah I see, this was added in the TSDoc repo (https://github.com/microsoft/tsdoc/pull/317).